subprojects/degrees-and-programs/source/static/README.md

```diff
@@ -0,0 +1,57 @@
+# chapman
```

Comment:
This README is an automatically generated file as part of our generator. It's mostly for internal use – I can remove it if you'd like. The README that I wrote for you is located at `subprojects/degrees-and-programs/`.